packages/salesforcedx-visualforce-language-server/test/jest/formatting.test.ts

```diff
@@ -5,6 +5,7 @@
 'use strict';
 
 import * as assert from 'node:assert';
+// eslint-disable-next-line no-restricted-imports
```

Comment:
I think we should allow anything in `test` to import from node:fs for things like this code is doing with fixtures.

There's examples of that in our eslint rules (where you make something an error in the main part of the rules, and then turn the rules off for test stuff)